packages/collections/sort.pony

```diff
@@ -1,6 +1,21 @@
 primitive Sort[A: Seq[B] ref, B: Comparable[B] #read]
   """
   Implementation of dual-pivot quicksort.
+  
+  ## Example program
+  The following takes an reverse-alphabetical array, and sorts it alphabetically using the default String Comparator.
+  
+  ```pony
+     use "collections"
+
+     actor Main 
+       new create(env:Env) => 
+         let array = [ "third"; "second"; "first" ]
+         let sorted_array = Sort[Array[String], String](array)
+         for e in sorted_array.values() do
+           env.out.print(e)
+         end
+    ```
```

Comment:
For examples, it's good to tell people what the output is. For example here, have something like:

---

Would output:

```bash
first
second
third
```

Reply:
👍